rtb/norme_di_progetto_ver0.6.0.typ

```diff
@@ -531,9 +556,177 @@ Il verificatore può consultare sia il documento in formato `.pdf` per maggiore
+
+===== Verbali Esterni
+Per quanto riguarda le riunioni con i proponenti, valgono le stesse regole di quelle interne, con il passaggio aggiuntivo, di cui si occupa il responsabile, di far firmare il verbale ai proponenti una volta redatto e approvato.
+
+== Miglioramento
+=== Descrizione
+Secondo lo standardG ISO/IEC 12207:1995, il processo di miglioramento nel ciclo di vita del software è finalizzato a stabilire, misurare, controllare e migliorare i processi che lo compongono. L’attività di miglioramento è composta da:
+- Analisi: Identificare le aree di miglioramento dei processi;
+- Miglioramento: Implementare le modifiche necessarie per migliorare i processi di sviluppo del software;
+
+==== Analisi dei processi
+Innanzitutto occorre stabilire una serie di processi organizzativi per l'intero ciclo di vita del software applicabili alle varie attività di progetto. Quest'ultimi devono essere documentati, come viene fatto proprio qui, e va implementato un meccanismo di controllo per sviluppare, monitorare e migliorare i processi stessi. Per garantire efficacia e continuo miglioramento dei processi il responsabile guiderà una retrospettiva di gruppo da effettuare alla fine di ogni sprint e da mettere nero su bianco per ogni sprint nel documento Piano di Progetto.
+
+==== Miglioramento dei processi
+Una volta identificati i potenziali miglioramenti questi vanno effettivamente implementati secondo le corrette a adeguate metodologie. Nella retrospettiva seguente all'applicazione di queste migliorie verrà effettuato un altro _#rifGlossario("feedback")_ a riguardo per progredire ciclicamente.
+
+=== Metriche
+// TODO: Bisogna capire bene quali sono nostre metriche, aspettando il PdQ
+
+== Formazione
+=== Descrizione
+Alla base della buona riuscita del progetto ci sono persone adeguatamente formate a mano a mano che il progetto evolve. Per questo diventa fondamentale il processo di formazione.
+Con esso vengono definiti gli standard e le metodologie con i quali ognuno sarà sempre pronto nell'argomento adeguato nel momento opportuno.
+
+=== Descrizione
+Per iniziare a formare i membri del gruppo, è necessario prima comprendere completamente il dominio del problema. Quindi, è necessario comprendere quali argomenti sono necessari per essere approfonditi e quali abilità sono necessarie per i vari processi. \
+Si deve quindi passare all'individuazione del materiale di formazione, che aumenterà nel tempo poiché il nostro livello di conoscenza e comprensione del problema dovrà aumentare man mano che il progetto avanza.\
+Infine, tutti i membri del gruppo devono imparare da dove e cosa studiare, e devono anche aggiornarsi individualmente, se possibile, con l'aiuto di altri membri più esperti.
+
+=== Strumenti
+Al fine di agevolare il processo di formazione, ogni membro del gruppo incaricato in un determinato sprint di approfondire un determinato argomento dovrà essere in grado di tramandare la propria conoscenza agli altri. 
```

Comment:
- "sprint" da mettere in corsivo
- leverei la maiuscola a "Deep"

Reply:
Deep dive va in maiuscolo perché è nel glossario, se non sbaglio. Verifica e nel caso vada bene così risolvi pure la discussione

Reply:
perfetto, risolto